### PR TITLE
[fix/LIVE-13101]: ensure correct trx fees for trx and trc20 swaps and sends

### PR DESCRIPTION
**Pull request by kallen-ledger**
### ✅ Checklist

- [x] `npx changeset` was attached.
- [ ] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - ...

#### TRX Swap
https://github.com/LedgerHQ/ledger-live/assets/132384348/56c5306d-c90d-45c9-af19-b6ffd390883c

#### TRC20 Swap
https://github.com/LedgerHQ/ledger-live/assets/132384348/b70bf616-878b-4944-9f1d-58a02b7988fc

#### TRC20 Not enough balance error message
https://github.com/LedgerHQ/ledger-live/assets/132384348/018b5873-1a32-4238-a79a-51e6aa89c820

### ❓ Context

https://ledgerhq.atlassian.net/browse/LIVE-13101

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Comment by cng-ledger** [comment]
Looks good to me, might need to give CI a few retries tho

**Review by cng-ledger — APPROVED** [review]

**Review by cng-ledger — COMMENTED** [review]
👏

**Review by cng-ledger — APPROVED** [review]

File: libs/coin-modules/coin-tron/src/bridge/getEstimateFees.ts [hunk 1]
```diff
@@ -24,23 +24,48 @@ const getFeesFromBandwidth = (account: Account, transaction: Transaction): BigNu
 };
 
 // Special case: If activated an account, cost around 0.1 TRX
-const getFeesFromAccountActivation = async (a: Account, t: Transaction): Promise<BigNumber> => {
-  const recipientAccount = await fetchTronAccount(t.recipient);
+const getFeesFromAccountActivation = async (
+  a: Account,
+  t: Transaction,
+  ta?: TokenAccount | null,
```
Comment (lvndry) [inline libs/coin-modules/coin-tron/src/bridge/getEstimateFees.ts:30]: Use more understandable variable names:

```suggestion
  account: Account,
  transaction: Transaction,
  tokenAccount?: TokenAccount | null,
```

File: libs/coin-modules/coin-tron/src/bridge/getEstimateFees.ts [hunk 2]
```diff
@@ -24,23 +24,48 @@ const getFeesFromBandwidth = (account: Account, transaction: Transaction): BigNu
...
   }
 
   return new BigNumber(0); // no fee
 };
 
-const getEstimatedFees = async (a: Account, t: Transaction, isContract: boolean) => {
+const getEstimatedFees = async (
+  a: Account,
+  t: Transaction,
+  _isContract: boolean,
```
Comment (lvndry) [inline libs/coin-modules/coin-tron/src/bridge/getEstimateFees.ts:64]:
```suggestion
```

variable can be removed if not used

File: libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts [hunk 3]
```diff
@@ -17,8 +17,8 @@ const abandonSeedAddresses: Partial<Record<CryptoCurrency["id"], string>> = {
   ripple: "rHsMGQEkVNJmpGWs8XUBoTBiAAbwxZN5v3",
   stellar: "GDYPMQMYW2JTLPWAUAHIDY3E4VHP5SGTFC5SMA45L7ZPOTHWQ2PHEW3E",
   tezos: "tz1VJitLYB31fEC82efFkLRU4AQUH9QgH3q6",
-  // https://developers.tron.network/reference/getburntrx-1
-  tron: "TLsV52sRDL79HXGGm9yzwKibb6BeruhUzy",
+  // Randomly generated inactive tron address.
```
Comment (lvndry) [inline libs/ledgerjs/packages/cryptoassets/src/abandonseed.ts:20]: I think it was nice to know how it was generated. The link seems to still be relevant

**Review by lvndry — CHANGES_REQUESTED** [review]

**Review by lvndry — APPROVED** [review]

**Review by cng-ledger — APPROVED** [review]

**Pull request merged**